Vizzuality/half-earth-v3
# Develop

## Merge develop change into main
### Description
Merge NRC Analyze Area layer fix

### Testing instructions
Access NRC
Select and Explore a country
Click the "Go To Analyze Area" button
Verify that country information loads correctly
